sdk/schedules/schedule.ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
Nit: may rename this sample to "recurrence-schedule"?

Reply:
I think Han means "schedule-with-pipeline-job", in case we will add 'schedule-with-endpoint', etc.

Reply:
renamed